### PR TITLE
New PR based on external contribution

### PR DESCRIPTION
**Pull request by bgravenorst**
Based on https://github.com/Consensys/doc.linea/pull/594

This fixes up cspell issues and includes edits for the external contribution.

### CODE REVIEW

**Comment by jlwllmr** [comment]
Am I right in thinking this replaces #575? If so, we should close that PR before proceeding here

**Comment by bgravenorst** [comment]
> Am I right in thinking this replaces #575? If so, we should close that PR before proceeding here

No, I'll delete this one, since the user may have made additional updates not included here.

**Comment by bgravenorst** [comment]
This was for testing purposes only.

**Comment by bgravenorst** [comment]
@jlwllmr, I was originally going to close this PR. But I think it would be best to update this and close the original. I'll also edit the content in this PR.

File: project-words.txt
Comment (jlwllmr) [inline project-words.txt:1]: This should be alphabetical. I'll try and fix.

**Review by jlwllmr — COMMENTED** [review]
Need to adjust the ordering of words added to `project-words.txt`, as the alphabetical order has been broken.

**Review by jlwllmr — APPROVED** [review]

**Pull request merged**